testing/environments/local.yml

```diff
@@ -1,26 +0,0 @@
-# Defines if ports should be exported.
-# This is useful for testing locally with a full elastic stack setup.
-# All services can be reached through localhost like localhost:5601 for Kibana
-# This is not used for CI as otherwise ports conflicts could happen.
-version: '2.3'
-services:
-  kibana:
-    ports:
-      - "127.0.0.1:5601:5601"
-    depends_on:
-      elasticsearch:
-        condition: service_healthy
-
-  elasticsearch:
-    ports:
-      - "127.0.0.1:9200:9200"
-
-  package-registry:
-    build: ../..
```

Comment:
Is there still some test or something that tries to build the registry container?

Reply:
Currently, it's only in the Jenkinsfile: https://github.com/elastic/package-registry/blob/master/.ci/Jenkinsfile#L164

Reply:
I think this is enough, I only wanted to check that there is something checking that the container image can be built.